### ISSUE TITLE
Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

### ISSUE THREAD

**Issue by alanking** [irods/irods_development_environment#35]
Hadolint is recommending pinning versions for package installations, including pip packages. Is this something we should do?

**Comment by korydraughn** [comment]
i don't think this matters much for the development environment. each build is already restricted to the packages available by that OS distribution. if a dependency changes, we simply update the development environment and keep it moving.

**Comment by trel** [comment]
i think i agree - this is already designed around feeding it what we want it to use at the package level.

**Comment by alanking** [comment]
Sounds good. In that case, we will need to figure out how to disable that error in Codacy
